catalyst/core/callbacks/optimizer.py

```diff
@@ -217,8 +221,17 @@ def on_batch_end(self, runner: IRunner) -> None:
                 optimizer_wds=self._optimizer_wd,
                 grad_clip_fn=self.grad_clip_fn,
             )
+            if not self.use_fast_zero_grad:
+                utils.maybe_recursive_call(self._optimizer, "zero_grad")
+            else:
 
-            utils.maybe_recursive_call(self._optimizer, "zero_grad")
+                def zero_grad(optimizer):
+                    for group in optimizer.param_groups:
+                        for p in group["params"]:
+                            if p.grad is not None:
+                                p.grad = None
```

Comment:
@Scitator 
how do you think, should i move this function to utils ?

Reply:
yup, could we move to function on top on OptimizerCallback?

Reply:
thanks, done